content/case-studies/lishu-maharjan.md

```diff
@@ -9,6 +9,6 @@ company_logo: "https://rumsan.com/"
 content_heading: "Harnessing Potential: Lishu Maharjan’s TEJ Fellowship Journey"
 ---
 
-Lishu Maharjan, from Kathmandu's Newa community, is the eldest of three siblings. Her father runs a small furniture business, and her mother manages the household. Despite earning her Bachelor's degree in information management from Prime College, Lishu struggled with understanding coding and lacked confidence in asking questions during her classes. This became evident during an individual project where she managed the front-end development but felt lost when it came to the back end.
-Lishu discovered the TEJ Fellowship through her cousin and was motivated by its hands-on learning opportunities and structured program. Her goals included improving her technical skills, building networks, and gaining the confidence to become a better learner. Initially shy and anxious about asking questions, the mentorship and supportive environment at TEJ helped her overcome these challenges. Weekly presentations and interactions with her peers further built her confidence in both coding and communication.
+Lishu Maharjan, from Kathmandu's Newa community, is the eldest of three siblings. Her father runs a small furniture business, and her mother manages the household. Despite earning her Bachelor's degree in information management from Prime College, Lishu struggled with understanding coding and lacked confidence in asking questions during her classes. This became evident during an individual project where she managed the front-end development but felt lost when it came to the back end.<br/><br/>
```

Comment:
Can we make change at code level so that we do not have to add <br> to ALL the markdown files?